virtme/guest/virtme-init

```diff
@@ -236,42 +238,42 @@ real_sudoers=/etc/sudoers
 if [ ! -e ${real_sudoers} ]; then
     touch ${real_sudoers}
 fi
-tmpfile="`mktemp --tmpdir=/run/tmp`"
-echo "Defaults secure_path=\"/usr/sbin:/usr/bin:/sbin:/bin\"" > $tmpfile
-echo "root ALL = (ALL) NOPASSWD: ALL" >> $tmpfile
+tmpfile="$(mktemp --tmpdir=/run/tmp)"
+echo "Defaults secure_path=\"/usr/sbin:/usr/bin:/sbin:/bin\"" > "$tmpfile"
+echo "root ALL = (ALL) NOPASSWD: ALL" >> "$tmpfile"
 if [[ -n "${virtme_user}" ]]; then
-    echo "${virtme_user} ALL = (ALL) NOPASSWD: ALL" >> $tmpfile
+    printf -- '%s\n' "${virtme_user} ALL = (ALL) NOPASSWD: ALL" >> "$tmpfile"
```

Comment:
maybe clearer with:

    printf -- '%s ALL = (ALL) NOPASSWD: ALL\n' "${virtme_user}"

while switching to `printf`, better to use it as expected :)

Reply:
Yep - I agree.